compliance-by-design-demo/server/app.js

```diff
@@ -0,0 +1,44 @@
+const fetch = require("node-fetch")
+
+const express = require('express')
+const app = express()
+const port = 3001
+
+const redis = require("async-redis");
+const redis_kvk_cache = redis.createClient(
+    process.env.REDIS_KVK_CACHE_PORT,
+    process.env.REDIS_KVK_CACHE_HOST
+);
+
+redis_kvk_cache.on('connect', function() {
+    console.log('Redis client connected');
+});
+redis_kvk_cache.on("error", function(error) {
+    console.error(error);
+});
+
+const url = "api.kvk.nl"
+const endpoint = "/api/v2/testprofile/companies"
+
+async function fetchApi (endpoint) {
+    return fetch("https://" + endpoint)
+        .catch(err => { console.log('caught', err.message); return {"error": 1} })
+        .then(data => data.json())
+}
+
+app.get('/api/companies/:kvknr', async (req, res) => {
+    // TODO: req.params.kvknr validation
+    const kvknr = req.params.kvknr
+    const cacheval = await redis_kvk_cache.get(kvknr)
+    if (cacheval) {
+        console.log('Cache hit: ' + kvknr);
```

Comment:
These logs on every request are probably a little bit overkill and could eat more resources than we'd like.

Reply:
For now the "Cache miss" line is still in there for debugging. We might want some logging in production to get statistics, but we will resolve that later if needed.

Reply:
This is removed.